sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
       ],
     },
   ],
+
+  // Japanese sidebar nav
+  docsJapanese: [
+    {
+      type: 'doc',
+      label: 'ScalarDB ドキュメンテーションホーム',
+      id: 'index',
+    },
+    {
+      type: 'category',
+      label: 'ScalarDB について',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'database-configurations',
+          label: '基本の設定',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-notes',
+          label: 'リリースノート',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'リリースサポートポリシー',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'クイックスタート',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'quick-start-overview',
+          label: '概要',
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Core ライブラリを使用してトランザクションを実行',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb',
+              label: 'ScalarDB Core を使用',
+            },
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb-by-using-kotlin',
+              label: 'Kotlin を使用',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Cluster を使用してトランザクションを実行',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster',
+              label: 'ScalarDB Cluster を使用',
```

Comment:
The upper level says `ScalarDB Cluster` and all the contents are about ScalarDB Cluster, so I think it's a bit weird to say as above.
```suggestion
              label: 'Java API を使用',
```
I think the original English one should be 'Use Java API` or something.